sensor_msgs/PointCloud2 pointcloud
sensor_msgs/Image rgb_image
sensor_msgs/Image depth_image
sensor_msgs/Image mask
